include/sharg/concept.hpp

```diff
@@ -0,0 +1,54 @@
+// -----------------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2021, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2021, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/sharg-parser/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ * \brief Provides helper concepts.
+ */
+
+#pragma once
+
+#include <seqan3/std/concepts>
+
+#include <sharg/auxiliary.hpp>
+
+namespace sharg
+{
+
+/*!\concept sharg::istreamable
+ * \ingroup argument_parser
+ * \brief Concept for types that can be parsed from a std::istream via the stream operator.
+ * \tparam value_type The type to check whether it's stremable via std::istream.
+ *
+ * ### Requirements
+ *
+ * `std::istream` must support the (un)formatted input function (`operator>>`) for an l-value of a given `value_type`.
+ *
+ * todo: link to how-to page how to make your type istreamable
```

Comment:
What about this todo?

Reply:
Good point. 

The reason I don't do it right away is because it will change anyway once we also require `ostreamble` and thus I don't have to write the how-to twice.

I removed the todo here for now and added it instead to the issue: https://github.com/seqan/sharg-parser/issues/41